tests/unit/r2i/tflite/meson.build

```diff
@@ -0,0 +1,37 @@
+subdir('resources')
+
+lib_tflite_tests = [
+  ['engine.cc'],
+  ['frame.cc'],
+  ['loader.cc'],
+  ['parameters.cc'],
+  ['prediction.cc'],
+]
+
+# Build and run tests
+foreach t : lib_tflite_tests
+  fname = t[0]
+  test_name = fname.split('.')[0].underscorify()
+  has_flags = t.get(1,false)
+  flags = t.get(2,'')
+
+  if has_flags
+    test_flags = c_args + [flags]
+  else
+    test_flags = c_args
+  endif
+
+  exe = executable(test_name, fname,
+      c_args : test_flags,
+      cpp_args : test_flags,
+      include_directories : [configinc, r2i_inc_dir],
+      dependencies : [r2inference_lib_dep, tests_dep, lib_tflite_dep],
+  )
+
+  # Run tests
+  test(test_name, exe,
+       args : '-p',
+       timeout : 60,
+       workdir : meson.current_build_dir())
```

Comment:
Why don't we set `source_dir` here so we don't need to copy the models.

Reply:
I preferred to copy the resources. If you want to run the tests standalone (./engine for example), the tests will fail because the resources directory is not there at the same level of the test (as it does with autotools).